### ISSUE TITLE
parser not resolving remote reference pathItem 

### ISSUE THREAD

**Issue by gracekarina** [swagger-api/swagger-parser#1103]
parser not resolving remote reference pathItem

**Comment by gracekarina** [comment]
Swagger.yaml
```
openapi: 3.0.0
info:
  version: 2.0.0
  title: Admin Web App Services
  description: 'Services API'
paths:
  '/Translation/{lang}':
    $ref: './domain.yaml#/components/pathitems/Translation'
````

Domain.yaml

`````
openapi: 3.0.0
info:
  version: 1.0.0
  title: API Reuse Common Components
  description: Definitions of API components to be referred across different API files
components:
  pathitems:
    Translation:
      put:
        tags:
          - Translation
        summary: Set value of this parameter
        description: Provide the lookup table for localized strings
        parameters:
          - $ref: '#/components/parameters/param_lang'
  parameters:
    param_lang:
      name: lang
      in: path
      description: language - 2-letter abbreviation
      required: true
      schema:
        type: string
        enum:
          - en
          - se
          - dk
          - fi
          - false
          - de
          - es
          - et
          - fr
`````

**Comment by gracekarina** [comment]
closed by #1104
